docs:#632 single quotes around passwords (#922)
redpanda-data/documentation: docs/platform/reference/redpanda-operator/kubernetes-sasl.mdx

You must create the superuser through the Admin API. This user has [ALL permissions](../../../security/authorization#operations) on the cluster and is the user that grants permissions to new users. Without a superuser, you can create other users, but you will not be able to grant them permissions to the cluster.

:::note

> Passwords are in single quotes to avoid conflicts with special characters. Ensure arguments are properly quoted.

Reviewer: ```suggestion
Ensure arguments are properly quoted for your shell if they contain special characters.
```

> Passwords are in single quotes to avoid conflicts with special characters. Enclosing characters in single quotes preserves the literal value of each character.

Reviewer: This is probably true in bash, unless the password contains a single quote. This applies to all strings passed as arguments in bash. The username could contain a single quote. Other shells are available.

Reviewer: We might prefer to say something a bit more generic, such "Ensure the arguments are properly quoted". 

That might be less useful;.anybody not using something that's essentially bash is likely to know how their shell does quoting.

```bash
kubectl exec -c redpanda <cluster_name>-0 -- rpk acl create --allow-principal User:<username> --operation create,describe --cluster \
--user <superuser_username> \
--password <'superuser_password'> \
--sasl-mechanism SCRAM-SHA-256
```

1. Optionally, you can use the superuser to grant permissions to a new user for a topic within the cluster. The following command grants `describe` privileges to a topic that doesn’t exist yet. In the next step, you create the topic that you reference in this command.

```bash
kubectl exec -c redpanda <cluster_name>-0 -- rpk acl create --allow-principal User:<username> --operation describe --topic <topic_name> \
--user <superuser_username> \

> --password <'superuser_password'> \

Reviewer: This kind of looks like the `<` is required, outside of the quoting. I think `'<superuser_password>'` might be clearer.
